src/components/Profile/MissionsProfile.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+const MissionsProfile = () => {
+  const missionsx = useSelector((state) => state.Missions.data);
+  const reservedMissions = missionsx.filter((mission) => mission.reserved);
+  return (
+    <div className="reserve-container">
+      <h1 className="profile-rockets-title">My Missions</h1>
```

Comment:
- [ ] You have two `<h1>` tags on the same page, which is not a good practice in your markup, remember that this tag is reserved for the main title of each page because search engines use them to index the page, so please consider following this standard! 🤓 🔧